core/primitives/src/bandwidth_scheduler.rs

```diff
@@ -37,6 +37,7 @@ impl BandwidthRequests {
     }
 }
 
+/// Version 1 for list of [`BandwidthRequest`]s.
```

Comment:
The phrasing suggests that `BandwidthRequest` gets later put into a list.


```suggestion
/// Version 1 of [`BandwidthRequest`].
```